docs/content/functioneel/zaken_api.md

```diff
@@ -1,157 +1,237 @@
+<li>Het relateren van een Informatieobject aan een Zaak wordt aangestuurd vanuit de Documenten-API.</li>
+</ul></td>
+</tr>
+</tbody>
+</table>
 
-| **Aspect**     | **Beschrijving** |
-|----------------|------------------|
-| Aanleiding     |                  |
-| Voorwaarde     |                  |
-| Gevolg         |                  |
-| Gegevens       |                  |
-| Specificaties  |                  |
-| Samenhang      |                  |
-| Bijzonderheden |                  |
-
-
-*Gegevens*
-
-[tabel van Sergei met de gegevens van de resource Klantcontacten]
-
-
-*Wijziginghistorie*
-
-| **Datum** | **User-story(’s)** | **Wijziging** |
-|-----------|--------------------|---------------|
-|           |                    |               |
+### Resource: Zaakobjecten
 
+<table>
+<tbody>
+<tr class="odd">
+<td><strong>Aspect</strong></td>
+<td><strong>Beschrijving</strong></td>
+</tr>
+<tr class="even">
+<td>Doel</td>
+<td><ul>
+<li>Opvragen van een overzicht van Objecten bij een Zaak.</li>
+<li>Opvragen van – de gegevens van - een specifiek Zaakobject die de relatie beschijft tussen een Object en een Zaak.</li>
+<li>Relateren van een Object aan een Zaak d.m.v. het toevoegen van een Zaakobject.</li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td>Gegevens</td>
+<td>Relatieklasse ZAAKOBJECT, inclusief relatie ‘ZAAK betreft OBJECT’.<br/>
+https://www.gemmaonline.nl/index.php/rgbz_2.0/doc/relatieklasse/zaakobject</td>
+</tr>
+<tr class="even">
+<td>OAS-specificaties</td>
+<td><a href="https://ref.tst.vng.cloud/zrc/api/v1/schema/#tag/zaakobjecten">https://ref.tst.vng.cloud/zrc/api/v1/schema/#tag/zaakobjecten</a></td>
+</tr>
+<tr class="odd">
+<td>Bijzonderheden</td>
+<td><ul>
+<li>Het verwijderen van Zaakobjecten wordt nog niet ondersteund. Overigens kunnen Zaakobjecten niet zomaar verwijderd worden, hooguit als ‘correctie’ of na verstrijken van de archiverings-vernietigingstermijn.</li>
+<li>Het aanpassen van Zaakobjecten wordt nog niet ondersteund.</li>
+</ul>
+</td>
+</tr>
+</tbody>
+</table>
+
+### Klantcontacten
+
+<table>
+<tbody>
+<tr class="odd">
+<td><strong>Aspect</strong></td>
+<td><strong>Beschrijving</strong></td>
+</tr>
+<tr class="even">
+<td>Doel</td>
+<td><ul>
+<li>Opvragen van een overzicht van alle Klantcontacten met per Klantcontact enkele gegevens.</li>
+<li>Opvragen van – de gegevens van - een specifiek Klantcontact.</li> 
+<li>Aanmaken van een Klantcontact en deze relateren aan een Zaak.</li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td>Gegevens</td>
+<td>https://www.gemmaonline.nl/index.php/rgbz_2.0/doc/objecttype/klantcontactOAS-specificaties BijzonderhedenHet verwijderen van aakobjecten wordt nog niet ondersteund.Het aanpassen van aakobjecten wordt nog niet ondersteund. Resource: Klantcontacten<strong>AspectBeschrijving</strong>DoelOpvragen van van alle lantcontactenvan een specifiek lantcontact. Aanmaken van een lantcontact en deze relateren aan aak.GegevensObjecttype KLANTCONTACT, inclusief relatie naar ZAAK.</td>
```

Comment:
hoop copy-paste fouten bij klantcontacten

Reply:
Dat is geen copy paste fout maar een foutje van Pandoc geweest. Ik had er al een boel uitgehaald maar deze ff gemist